src/security/owasp-zap/tasks/create-bugs.yaml

```diff
@@ -0,0 +1,118 @@
+# Create bugs for any failed tests
```

Comment:
As a general question, how will this handle the same issue coming back from multiple scans? The scenario I'm thinking about is a system that has multiple apps/APIs. If the same vulnerability is present in more than one app, will that result in multiple bugs (it should), or will it think they're duplicates of each other?

Reply:
Sorry just realised the scan name is already included in the repro steps - is that sufficient?

We could also include it in the bug title but one of the names from the example pipeline is `UI App Unauthenticated Baseline Scan` which would be quite long in the title along with the OWASP Alert name?

Reply:
Would that mean extending the query to get existing bugs to include something like `Microsoft.VSTS.TCM.ReproSteps CONTAINS $($scanName)`?